lib/internal/promisify.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Thanks for the second problem; fixed.

I don't think promisify should automatically work on things with more than one argument, personally. A variant would be better.